utils/wipe/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wipe
+PKG_VERSION:=0.24
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/berke/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=aefb4403333562f9af5e3e03ecbba4b124e98788c688662240f9b04d80bfdb2f
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/wipe
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=System and process monitor for Linux
+  URL:=http://lambda-diode.com/software/wipe/
+  MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
```

Comment:
This one is duplicated. You have it already stated on line 12. Remove this one.

Reply:
done